src/app/CreateRecording/CustomRecordingForm.tsx

```diff
@@ -243,6 +284,72 @@ export const CustomRecordingForm = (props) => {
           </SplitItem>
         </Split>
       </FormGroup>
+      <ExpandableSection toggleTextExpanded="Hide advanced options" toggleTextCollapsed="Show advanced options">
+        <Form>
+          <Text component={TextVariants.small}>
+            A value of 0 for maximum age or size means unbounded.
```

Comment:
I definitely think we need some description/explanation for users about the difference between a recording's duration and the maximum age option - at face value they sound like they would do the same thing.

Reply:
I added helper text, but I'm not sure if its explains it correctly. Let me know if there's a better way to phrase it.

Reply:
I think the explanation is phrased fine. Maybe now we just need some description of what exactly the "duration" of a recording is then - ie explain that maxAge is the maximum age of the data kept in the buffer, and duration specifies when the recording will be automatically stopped.

Reply:
Ok, I added helper text for duration as well.